.idea/misc.xml

```diff
@@ -1,5 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="../../../../../layout/compose-model-1631723355678.xml" value="0.67" />
+        <entry key="../../../../../layout/compose-model-1631731131942.xml" value="0.5638888888888889" />
+        <entry key="../../../../../layout/compose-model-1631886742866.xml" value="0.1" />
+        <entry key="../../../../../layout/compose-model-1631908005804.xml" value="1.0824074074074075" />
+        <entry key="Demo/src/main/res/layout/activity_demo.xml" value="0.325" />
+        <entry key="Demo/src/main/res/layout/fragment_pay_pal.xml" value="0.3671875" />
+        <entry key="Demo/src/main/res/xml/root_preferences.xml" value="0.32382246376811596" />
+      </map>
+    </option>
+  </component>
```

Comment:
q: what does this file do? I'm never sure what from the `.idea` folder should be kept in the repo.

Reply:
@sarahkoop and I were asking the same question. Jetbrains recommends keeping most files in the .idea directory in source control. https://intellij-support.jetbrains.com/hc/en-us/articles/206544839

Although I'm not exactly sure what the misc.xml file is used for. It seems that additional tags are added when resources are added to the project along with new composables.

Reply:
Yeah it does kind of remind me of Xcode storyboards a bit. It jumped out during the review I'm fine with leaving as well.